docs/Building_Diagrams/escalation_events.md

```diff
@@ -88,4 +88,45 @@ In this scenario, we observe an End Escalation following a Boundary Event.
 This thrown End Escalation can be intercepted by the boundary event of a Call Activity or at the start of a sub-process, as shown in previous examples on this page.
 In another example, we see an End Escalation event being thrown within a sub-process and caught at the sub-process's boundary.
 The application of the last example aligns with the first, where the escalation can be intercepted either on a Call Activity or at the beginning of a sub-process.
-It's crucial to remember that whether a process is created or terminated in these contexts depends on whether non-interrupting or interrupting events are utilized.
\ No newline at end of file
+It's crucial to remember that whether a process is created or terminated in these contexts depends on whether non-interrupting or interrupting events are utilized.
+
+## Configuring Escalation Events Properties
+Setting up an escalation event within a workflow in SpiffWorkflow involves defining both the escalation trigger (throw event) and the point where the escalation is handled (catch event). 
+
+Here's how to set up these components:
+
+**Define the Escalation ID**: Determine the task or process stage where an escalation might need to be triggered due to exceptional circumstances. and decide on a unique identifier for your escalation event.
```

Comment:
maybe just a comma here: "exceptional circumstances. and decide on a unique identifier"

Reply:
Done